scripts/start-miden-node.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+miden-node make-genesis --inputs-path genesis.toml
```

Comment:
I think we can generate the genesis in the image generation and cache it.

IOW, this should be a `RUN` directive in the `Dockerfile` instead of being here

Reply:
Indeed, changing it.